language/stdlib/modules/Vector.move

```diff
@@ -72,8 +76,8 @@ module Vector {
         false
     }
 
-    // Return (true, i) if `e` is in the vector `v` at index `i`.
-    // Otherwise returns (false, 0).
+    /// Return `(true, i)` if `e` is in the vector `v` at index `i`.
+    /// Otherwise, returns `(false, 0)`.
     public fun index_of<Element>(v: &vector<Element>, e: &Element): (bool, u64) {
```

Comment:
A hold-over from the non-option days...I'll file an issue to change this to an option.

Reply:
SGTM, although the fact that `Option` is implemented using `Vector` may make this tricky...

Reply:
oh....you're right...nevermind! I keep forgetting that our options aren't really options.